js/lykke2.js

```diff
@@ -0,0 +1,1171 @@
+                'type': type,
+                'name': name,
+                'active': active && deposit && withdraw,
+                'deposit': deposit,
+                'withdraw': withdraw,
+                'fee': undefined,
+                'precision': this.safeNumber (currency, 'accuracy'),
+                'limits': {
+                    'withdraw': {
+                        'min': this.safeValue (currency, 'cashoutMinimalAmount'),
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': this.safeValue (currency, 'lowVolumeAmount'),
+                        'max': undefined,
+                    },
+                },
+            };
+        }
+        return result;
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetAssetpairs (params);
+        const markets = this.safeValue (response, 'payload', []);
+        //
+        //     {
+        //         "payload":[
+        //             {
+        //                 "assetPairId":"AAVEBTC",
+        //                 "baseAssetId":"c9e55548-dae5-44fc-bebd-e72249cb19f3",
+        //                 "quoteAssetId":"BTC",
+        //                 "name":"AAVE/BTC",
+        //                 "priceAccuracy":6,
+        //                 "baseAssetAccuracy":6,
+        //                 "quoteAssetAccuracy":8,
+        //                 "minVolume":0.001,
+        //                 "minOppositeVolume":0.0001
+        //             }
+        //         ],
+        //         "error":null
+        //     }
+        //
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'assetPairId');
+            const name = this.safeString (market, 'name');
+            const [ baseId, quoteId ] = name.split ('/');
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const symbol = base + '/' + quote;
+            const pricePrecision = this.safeString (market, 'priceAccuracy');
+            const priceLimit = this.parsePrecision (pricePrecision);
+            const precision = {
+                'price': parseInt (pricePrecision),
+                'amount': this.safeInteger (market, 'baseAssetAccuracy'),
+            };
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'settle': undefined,
+                'settleId': undefined,
+                'type': 'spot',
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'derivative': false,
```

Comment:
we don't need this field derivative anymore, we've since removed it from the lub